utils/build.ps1

```diff
@@ -1022,26 +1110,30 @@ function Build-Compilers() {
 
     Build-CMakeProject `
       -Src $SourceCache\llvm-project\llvm `
-      -Bin (Get-HostProjectBinaryCache Compilers) `
+      -Bin $CompilersBinaryCache `
       -InstallTo "$($Arch.ToolchainInstallRoot)\usr" `
       -Arch $Arch `
       -UseMSVCCompilers C,CXX `
+      -UsePinnedCompilers Swift `
       -BuildTargets $Targets `
       -CacheScript $SourceCache\swift\cmake\caches\Windows-$($Arch.LLVMName).cmake `
       -Defines ($TestingDefines + @{
         CLANG_TABLEGEN = (Join-Path -Path $BuildTools -ChildPath "clang-tblgen.exe");
         CLANG_TIDY_CONFUSABLE_CHARS_GEN = (Join-Path -Path $BuildTools -ChildPath "clang-tidy-confusable-chars-gen.exe");
-        CMAKE_Swift_COMPILER = (Join-Path -Path (Get-PinnedToolchainTool) -ChildPath "swiftc.exe");
-        CMAKE_Swift_FLAGS = @("-sdk", (Get-PinnedToolchainSDK));
         LLDB_PYTHON_EXE_RELATIVE_PATH = "python.exe";
         LLDB_PYTHON_EXT_SUFFIX = ".pyd";
         LLDB_PYTHON_RELATIVE_PATH = "lib/site-packages";
         LLDB_TABLEGEN = (Join-Path -Path $BuildTools -ChildPath "lldb-tblgen.exe");
         LLVM_CONFIG_PATH = (Join-Path -Path $BuildTools -ChildPath "llvm-config.exe");
         LLVM_EXTERNAL_SWIFT_SOURCE_DIR = "$SourceCache\swift";
+        LLVM_INCLUDE_BENCHMARKS = "NO";
```

Comment:
why do we need not to include the benchmarks now?

Reply:
This turned out to be unnecessary. Reverted.

Reply:
I think that this might be needed still - this avoids unnecessary checks and configuration. I can do the change separately, but the problem was that one of the checks required execution which would not work with cross-compiling.

Reply:
That may be right. When I rerun without these changes, the build worked successfully.